### PR TITLE
add support for flake-compat

### PR DESCRIPTION
**Pull request by riceicetea**
works on my machine

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -7,9 +7,13 @@
...
     rust-overlay.inputs.flake-utils.follows = "flake-utils";
     rust-overlay.inputs.nixpkgs.follows = "nixpkgs";
+    flake-compat = {
+      url = "github:edolstra/flake-compat";
+      flake = false;
+    };
   };
 
-  outputs = { self, flake-utils, nixpkgs, rust-overlay }:
+  outputs = { self, flake-utils, nixpkgs, rust-overlay, ...}:
```
Comment (oxalica) [inline flake.nix:16]:
```suggestion
  outputs = { self, flake-utils, nixpkgs, rust-overlay, ... }:
```

**Review by oxalica — CHANGES_REQUESTED** [review]
Other diff LGTM.

**Comment by oxalica** [comment]
Also please capitalize your commit message.

**Comment by pleshevskiy** [comment]
I don't like an idea to add `flake-compat` to input. I use flakes and I don't want to download `flake-compat`. But we can use [this template](https://github.com/NixOS/templates/tree/master/compat)
```
nix flake init -t templates#compat
```

**Pull request closed without merging**